apps/src/code-studio/initApp/project.js

```diff
@@ -1178,6 +1180,13 @@ var projects = (module.exports = {
               if (saveSourcesErrorCount >= NUM_ERRORS_BEFORE_WARNING) {
                 header.showTryAgainDialog();
               }
+              if (err.message.includes('httpStatusCode: 422')) {
+                const msg =
+                  'There was an error saving your project, please remove any invalid characters to resolve.';
```

Comment:
We should add this as a string key so it can be translated. You can add a key-value mapping for this in apps/i18n/common/en_us.json (such as `"invalidCharactersErrorMessage": "There was an error..."`), and then reference that key here (`msg.invalidCharactersErrorMessage()`). Looks like the string object has already been imported in this file as `msg`.